src/neo/SmartContract/Iterators/PrimitiveWrapper.cs

```diff
@@ -0,0 +1,41 @@
+using Neo.VM.Types;
+using System;
+
+namespace Neo.SmartContract.Iterators
+{
+    internal class PrimitiveWrapper : IIterator
```

Comment:
I think this is essentially a ByteArrayWrapper. See the next line:

```c#
private readonly byte[] array;
```

Reply:
But we accept Integer because is primitive

Reply:
But we are equivalent to doing an implicit conversion.

Reply:
Ok I will rename it again